pg-index-health-testing/src/main/java/io/github/mfvanek/pg/testing/PostgreSqlContainerWrapper.java

```diff
@@ -39,7 +40,8 @@ public final class PostgreSqlContainerWrapper implements AutoCloseable, Postgres
                 .withTag(pgVersion.getVersion()))
                 .withSharedMemorySize(MemoryUnit.MB.convertToBytes(512))
                 .withTmpFs(Map.of("/var/lib/postgresql/data", "rw"))
-                .withCommand(prepareCommandParts(additionalParameters));
+                .withCommand(prepareCommandParts(additionalParameters))
+                .waitingFor(Wait.defaultWaitStrategy());
```

Comment:
@Evreke it's pretty interesting https://www.xavierbouclet.com/2022/08/08/TestContainer-Wait-DB-SQL.html

Reply:
How did you found it? We can implement such thing on cluster wrapper.